learn/advanced/filtering_and_faceted_search.md

```diff
@@ -18,7 +18,7 @@ Updating `filterableAttributes` requires recreating the entire index. This may t
 By default, `filterableAttributes` is empty. This means that filters do not work without first explicitly adding attributes to the `filterableAttributes` list.
 :::
 
-Filters work with numeric and string values. Empty fields or fields containing an empty array will be ignored.
+Filters work with numeric and string values. Empty fields or fields containing an empty array will be ignored. Filtering with `inf` or `NaN` as numbers will throw an error as they are [not supported by JSON](https://en.wikipedia.org/wiki/JSON#Data_types). Filtering with  `inf` or `NaN` as strings will work for all fields except [geo fields](/learn/advanced/geosearch.md#preparing-documents-for-location-based-search).  
```

Comment:
I'm not sure about `inf`—it's fairly obvious, I guess, but are all users going to understand it means `Infinity`?  Perhaps we can use `Infinity` or even just "infinite numbers"?